wiki/People/The_Team/Beatmap_Nominators/Becoming_a_Beatmap_Nominator/en.md

```diff
@@ -4,35 +4,67 @@ Users interested in joining the [Beatmap Nominators](/wiki/People/The_Team/Beatm
+  - Mod posts should explain the issue and the solution as clearly and concisely as possible to avoid confusing the mapper, or even having their suggestion applied incorrectly. Specialised terminology should also generally be avoided.
+
+## Application process
+
+![Application flowchart](img/ApplicationFlowchart.png "Flowchart outlining the Beatmap Nominator application process")
+
+### Individual stage
+
+During the individual stage, 2–4 evaluators are randomly assigned to the application. Sometimes, BNs are invited to be an evaluator, but there will always be at least one NAT assigned to the application.
+
+The evaluators will look over your application and evaluate your modding performance and your behaviour. They will typically comment down any positive and negative posts in the mods. Each evaluator will then vote "pass", "neutral", or "fail" for the application.
+
+Applicants will be evaluated on the following criteria:
+
+- **Modding:** Knowledge of the general [ranking criteria](/wiki/Ranking_Criteria) and specific criteria of each [game mode](/wiki/Game_mode). Adherence to the [modding expectations](#modding-expectations). Additional abilities relating to metadata, timing, and some others will also be taken into consideration.
+- **General attitude:** Tone of delivery of the mods and other behavioural expectations from the [Code of Conduct](/wiki/Rules/Code_of_Conduct_for_Modding_and_Mapping), as well as general behaviour in chat/forums.
+
+Any comments and votes from evaluators at this stage are hidden from other evaluators.
+
+### Group stage
+
+Applications will progress to the group stage when either:
+
+- All assigned evaluators have submitted their individual evaluation.
+- The majority of the evaluators have submitted their individual evaluation, where they have unanimously failed the applicant.
 
-### How new potential Beatmap Nominators are chosen
+At this stage, the comments and votes from each evaluator are revealed, and in the case of an ambiguous verdict, they will discuss until a consensus is reached.
 
-- Modders will be evaluated by the corresponding NAT members of the game mode they applied for, taking into account the following criteria:
-  - General attitude: Tone of delivery of the mods and other behavioural expectations from the [Code of Conduct](/wiki/Rules/Code_of_Conduct_for_Modding_and_Mapping), as well as general behaviour in chat/forums.
-  - Modding abilities: Knowledge of the general [Ranking Criteria](/wiki/Ranking_Criteria) and specific criteria of each game mode. Additional abilities like Metadata, Timing, and some others will also be taken into consideration.
-- After evaluating each modder, a discussion will be made on whether the modder will join the Beatmap Nominators.
+Feedback is then written based on the comments of the evaluators and the application result is sent back to the applicant. Successful applicants are added as a probationary BN immediately after the result is sent.
```

Comment:
am i tripping or isn't this just https://osu.ppy.sh/wiki/en/People/The_Team/Nomination_Assessment_Team/Evaluations#individual-phase.1? or are they just similar?

Reply:
as in the section over there should be deleted, with any relevant info moved over here